LongWarOfTheChosen/Src/LW_WeaponsAndArmor/Classes/X2Item_FactionWeapons.uc

```diff
@@ -1,5 +1,5 @@
 // This is an Unreal Script
-class X2Item_Weapons extends X2Item config(GameData_WeaponData);
+class X2Item_FactionWeapons extends X2Item config(GameData_WeaponData);

```

Comment:
I'm confused. Is this file needed still, since the weapon configuration is in the `ItemTable` array and the templates already exist?

Reply:
hmmm well this is actually a good point. I think it will still be needed unless I can somehow apply the weapon ability to the bullpups via the item table. If I can then I should be able to delete the class. If not then I will need to keep it so that I can apply the ability to the bullpups.

Reply:
As per our conversation on discord I am resolving this conversation as we are choosing to keep the templates.